linera-core/src/client/mod.rs

```diff
@@ -1740,7 +1739,8 @@ where
     /// Attempts to execute the block locally. If any incoming message execution fails, that
     /// message is rejected and execution is retried, until the block accepts only messages
     /// that succeed.
-    #[instrument(level = "trace", skip(block))]
+    // TODO[#2806]: Measure how failing messages affect the execution times.
```

Comment:
nit: We use `TODO(#xxxx)` in the rest of the code

Reply:
Done in [Review pt2](https://github.com/linera-io/linera-protocol/pull/2745/commits/1c2b92ece23d1d7623718d7ca723563505a14da8).